css/CornerWidgetUI.default.css

```diff
@@ -306,9 +311,10 @@ ul.country-list {
 	border: 2px solid #4067CB;
 	border-radius: 5px;
 	width: 150px;
-	height: 30px;
+	min-height: 30px;
```

Comment:
Should we also add a `min-width` and default content on an `:empty` selector? This will help prevent against empty text providing an odd looking bubble at the bottom of the dialog